tests/e2e/e2e_certmanager_test.go

```diff
@@ -17,9 +18,12 @@ var _ = OSMDescribe("1 Client pod -> 1 Server pod test using cert-manager",
 	},
 	func() {
 		Context("CertManagerSimpleClientServer", func() {
-			const sourceNs = "client"
-			const destNs = "server"
-			var ns []string = []string{sourceNs, destNs}
+			var (
+				clientNamespace     = framework.RandomName()
```

Comment:
I'm thinking have a function `RandomNameWithPrefix()` to allowing setting a prefix to the name. The purpose is that, sometimes we can debug on failed tests, and have a way to identify what's each ns for.

Reply:
Definitely went back and forth on that but this is a great idea. Thanks @allenlsy I'll update. cc/ @trstringer